app/Listeners/CheckoutableListener.php

```diff
@@ -60,14 +61,26 @@ public function onCheckedOut($event)
             $adminCcEmailsArray = array_map('trim', explode(',', $adminCcEmail));
         }
         $ccEmails = array_filter($adminCcEmailsArray);
-        $notifiable = $event->checkedOutTo;
+
         $mailable = $this->getCheckoutMailType($event, $acceptance);
+
+        if($event->checkedOutTo instanceof Asset){
+            $event->checkedOutTo->load('assignedTo');
+            $notifiable = $event->checkedOutTo->assignedto?->email ?? '';
+        }
+        else if($event->checkedOutTo instanceof Location) {
+            $notifiable = $event->checkedOutTo->manager?->email ?? '';
+        }
+        else{
+            $notifiable = $event->checkedOutTo->email;
```

Comment:
This is a good candidate for getting pulled into a `getNotifiable()` method. Not needed but would clear up some duplication and improve readability.

Reply:
💯 , Moved the logic to getNotifiables 👍